presto-main/src/test/java/com/facebook/presto/sql/planner/assertions/PlanMatchPattern.java

```diff
@@ -247,10 +277,150 @@ private String indentString(int indent)
         return Strings.repeat("    ", indent);
     }
 
-    private static class AnySymbolReference
-            extends SymbolReference
+    public static SymbolStem symbolStem(String stem)
     {
-        AnySymbolReference()
+        return new SymbolStem(stem);
+    }
+
+    /*
+     * All comments below about Symbols apply to SymbolReferences as well.
```

Comment:
For whoever reviews this: Nobody is delighted by this, least of all me. I'll go so far as to say it solves the problem at hand, but it sure isn't pretty. See notes below about some other ideas that might be worth trying if somebody has a clear preference.